devito/types/equation.py

```diff
@@ -236,15 +235,15 @@ def solve(eq, target, **kwargs):
         Symbolic optimizations applied while rearranging the equation. For more
         information. refer to ``sympy.solve.__doc__``.
     """
+    from devito.operations.solve import linsolve, SolveError
```

Comment:
imho, time to move `solve(...)` to `operations`

Reply:
done